.github/workflows/est-tests.yml

```diff
@@ -17,7 +17,30 @@ jobs:
   est-basic-test:
     name: Basic EST
     needs: [init, build]
-    uses: ./.github/workflows/est-basic-test.yml
-    with:
-      db-image: ${{ needs.init.outputs.db-image }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
 
+      - name: Retrieve PKI images
+        uses: actions/cache@v3
+        with:
+          key: pki-images-${{ github.sha }}
+          path: pki-images.tar
+
+      - name: Load PKI images
+        run: docker load --input pki-images.tar
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
```

Comment:
We could change it on backport? The Python version of `fedora:latest` is `3.11`, some of the Python code in this branch may have been updated to use languages features introduced in `3.7+`. That might be unlikely, but if this playbook conversion really takes off then whatever we do here will probably get cargo-culted around so we should be wary of what we choose.

Reply:
We already downgraded the `master` branch to Python 3.6 recently for downstream which involved a number of changes and some time to test. If we can maintain this level of compatibility, we won't need to do it again every time we create a new `v11.x` branch for downstream.

Reply:
Actually, this is the python version on the controller node which in this case is _ubuntu 22.04_ IIRC. When we test for different branch do we change the runner OS or just the containers? If the runner OS does not change then we do not need to modify the python version here. To be consistent with test made locally on Fedora, for whatever release, it could be important to use  the correct **ansible** release. Currently, it is used last available in _PyPi_ while in my local test I used the one available in fedora 38. I did not check if they are the same but we could force this.

Reply:
Ah ok, this is for running the playbook on GH so it's probably fine. It's more important to make sure that the playbook itself can run on RHEL 8 (with whatever Python/Ansible version it has) in case someone needs to run it locally.